praw/__init__.py

```diff
@@ -1032,9 +1040,15 @@ def get_submission(self, url=None, submission_id=None, comment_limit=0,
 
         """
         if bool(url) == bool(submission_id):
-            raise TypeError('One (and only one) of id or url is required!')
+            raise TypeError('One (and only one) of submision_id or '
```

Comment:
I see that you're just following the example, but can we drop the `!` in both cases? It seems harsh. A `.` will do better I think.